src/main/java/com/ericsson/ei/queryservice/ProcessQueryParams.java

```diff
@@ -68,12 +60,14 @@
      */
     public JSONArray runQuery(JSONObject criteriaObj, JSONObject optionsObj, String filter) {
         JSONArray resultAggregatedObject;
-        String criteria = editObjectNameInQueryParam(criteriaObj);
+        // String criteria = editObjectNameInQueryParam(criteriaObj);
+        String criteria = criteriaObj.toString();
 
         if (optionsObj == null || optionsObj.toString().equals("{}")) {
             resultAggregatedObject = processAggregatedObject.processQueryAggregatedObject(criteria, databaseName, aggregationCollectionName);
         } else {
-            String options = editObjectNameInQueryParam(optionsObj);
+            // String options = editObjectNameInQueryParam(optionsObj);
```

Comment:
I guess it's the same here, maybe remove this?

Reply:
removed